docs/queries/derivativeScrt.md

```diff
@@ -0,0 +1,75 @@
+# Scrt Derivative Examples
```

Comment:
update docs to include the function where you get staking info and fees together.